context.c

```diff
@@ -102,14 +102,21 @@ static ssize_t iio_snprintf_context_xml(char *ptr, ssize_t len,
 	ssize_t ret, alen = 0;
 	unsigned int i;
 
-	if (ctx->description)
-		ret = iio_snprintf(ptr, len, "%s<context name=\"%s\" "
-				   "description=\"%s\" >",
-				   xml_header, ctx->name, ctx->description);
-	else
-		ret = iio_snprintf(ptr, len, "%s<context name=\"%s\" >",
-				   xml_header, ctx->name);
+	ret = iio_snprintf(ptr, len, "%s<context name=\"%s\" ",
+			   xml_header, ctx->name);
+	if (ret < 0)
+		return ret;
 
+	iio_update_xml_indexes(ret, &ptr, &len, &alen);
+
+	if (ctx->description) {
+		ret = iio_xml_print_and_sanitized_param(ptr, len,
+							"description=\"",
+							ctx->description,
+							"\" >");
+	} else {
+		ret = iio_snprintf(ptr, len, ">");
+	}
```

Comment:
nit: just to keep consistency (maybe the rest of libiio?) with the change we are doing here, I would not use brackets for single line `if()...else`. I know, linux coding style biased

Reply:
I'm linux coding style biaised too, if you have multi-line statements in the "if" block, then you need brackets in the "else" even if there is only one line.

Reply:
Hmm, yes. You can actually have it both ways. As this is still the same line just splitted in multiple lines, you can still remove the brackets. Adding them is also fine... forget this